Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/OneClimbyBoi.java

```diff
@@ -0,0 +1,70 @@
+package org.usfirst.frc.team5493.robot.subsystems;
+
+import org.usfirst.frc.team5493.robot.Robot;
+import org.usfirst.frc.team5493.robot.RobotMap;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.CounterBase.EncodingType;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class OneClimbyBoi extends Subsystem {
+
+	public WPI_TalonSRX climbMotor;
+	public Object climbDown;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make climbDown a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)